scripts/receive_meteor.sh

```diff
@@ -176,59 +176,50 @@ if [ "$METEOR_RECEIVER" == "rtl_fm" ]; then
+  $METEORDEMOD -t "$TLE_FILE" -f jpg -i "${qpsk_file}" >> $NOAA_LOG 2>&1 #Dodao sam >> $NOAA_LOG 2>&1
 
-  rm "${qpsk_file}"
+  log "Removing QPSK, GCP and BMP files" "INFO"
+  rm "${qpsk_file}" *.gcp *.bmp
 
-  if [ -f "${AUDIO_FILE_BASE}.dec" ]; then
-    if [ "${SUN_ELEV}" -lt "${SUN_MIN_ELEV}" ]; then
-      log "I got a successful ${FILENAME_BASE}.dec file. Decoding APID 68" "INFO"
-      ${IMAGE_PROC_DIR}/meteor_apid68_decode.sh "${AUDIO_FILE_BASE}.dec" "${IMAGE_FILE_BASE}-122" >> $NOAA_LOG 2>&1
-      $CONVERT $FLIP -negate "${IMAGE_FILE_BASE}-122.bmp" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
-    else
-      log "I got a successful ${FILENAME_BASE}.dec file. Creating false color image" "INFO"
-      ${IMAGE_PROC_DIR}/meteor_false_color_decode.sh "${AUDIO_FILE_BASE}.dec" "${IMAGE_FILE_BASE}-122" >> $NOAA_LOG 2>&1
-	  $CONVERT $FLIP "${IMAGE_FILE_BASE}-122.bmp" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
-    fi
+  log "Annotating images and creating thumbnails" "INFO"
+  counter=1
+  for i in *.jpg
+    do
+      ${IMAGE_PROC_DIR}/meteor_normalize_annotate.sh "$i" "$i" 100 >> $NOAA_LOG 2>&1
+      ${IMAGE_PROC_DIR}/thumbnail.sh 300 "$i" "${i%.jpg}-thumb-122-rectified.jpg" >> $NOAA_LOG 2>&1
 
-    log "Rectifying image to adjust aspect ratio" "INFO"
-    python3 "${IMAGE_PROC_DIR}/meteor_rectify.py" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
+      mv "$i" "${IMAGE_FILE_BASE}-${counter}-122-rectified.jpg"
+      mv "${i%.jpg}-thumb-122-rectified.jpg" "${IMAGE_THUMB_BASE}-${counter}-122-rectified.jpg"
 
-    log "Annotating images and creating thumbnails" "INFO"
-    ${IMAGE_PROC_DIR}/meteor_normalize_annotate.sh "${IMAGE_FILE_BASE}-122-rectified.jpg" "${IMAGE_FILE_BASE}-122-rectified.jpg" 100 >> $NOAA_LOG 2>&1
-    ${IMAGE_PROC_DIR}/thumbnail.sh 300 "${IMAGE_FILE_BASE}-122-rectified.jpg" "${IMAGE_THUMB_BASE}-122-rectified.jpg" >> $NOAA_LOG 2>&1
-    rm "${IMAGE_FILE_BASE}-122.bmp"
-    rm "${AUDIO_FILE_BASE}.bmp"
-    rm "${AUDIO_FILE_BASE}.dec"
+      push_file_list="$push_file_list ${IMAGE_FILE_BASE}-${counter}-122-rectified.jpg "
+      ((counter++))
+    done
+    counter=1
 
-    if [ -f "${IMAGE_FILE_BASE}-122-rectified.jpg" ]; then
+    if [ -f "${IMAGE_FILE_BASE}-1-122-rectified.jpg" ]; then
       if [ "$ENABLE_EMAIL_PUSH" == "true" ]; then
         log "Emailing image" "INFO"
-        ${PUSH_PROC_DIR}/push_email.sh "${EMAIL_PUSH_ADDRESS}" "${IMAGE_FILE_BASE}-122-rectified.jpg" "${push_annotation}" >> $NOAA_LOG 2>&1
+	${PUSH_PROC_DIR}/push_email.sh "${EMAIL_PUSH_ADDRESS}" "${IMAGE_FILE_BASE}-2-122-rectified.jpg" "${push_annotation}" >> $NOAA_LOG 2>&1
       fi
 
       if [ "${ENABLE_DISCORD_PUSH}" == "true" ]; then
         log "Pushing image to Discord" "INFO"
-        ${PUSH_PROC_DIR}/push_discord.sh "${IMAGE_FILE_BASE}-122-rectified.jpg" "${push_annotation}" >> $NOAA_LOG 2>&1
+	${PUSH_PROC_DIR}/push_discord.sh "${IMAGE_FILE_BASE}-2-122-rectified.jpg" "${push_annotation}" >> $NOAA_LOG 2>&1
```

Comment:
This indentation change also looks wrong?

Reply:
Also filename changed.